source/macros.tex

```diff
@@ -214,6 +214,11 @@
 \newcommand{\libmember}[2]{\indexlibrarymember{#1}{#2}#1}
 \newcommand{\libspec}[2]{\indexlibrarymemberx{#1}{#2}#1}
 
+% index for macros
+% use in-place defining the macro in header synopses
+% initially just a global entry in the library index
```

Comment:
If you want to write an explanatory sentence, please make that a real sentence (full stop at the end, capital letter at the start).  For example

% index for macros
% Use where the macro is defined in the header synposis.

"% initially just a global entry in the library index"

Can go, I think; we don't want to commit to anything happening in the future.